### PR TITLE
Actus

### PR DESCRIPTION
**Pull request by KevinRoig**

### CODE REVIEW

**Review by AurelieFqt — DISMISSED** [review]

**Review by AurelieFqt — DISMISSED** [review]

**Review by AurelieFqt — DISMISSED** [review]

**Review by AurelieFqt — DISMISSED** [review]

File: src/Controller/ActualityController.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+class ActualityController extends AbstractController
+{
+    /**
+     * @Route("/actualites", name="actuality")
+     */
+    public function showArticles(ArticleRepository $articleRepository): Response
+    {
+        $articles = $articleRepository->findBy(
+            ["section" => 'Actualites'],
+            ['id' => 'DESC'],
```
Comment (RomainClair) [inline src/Controller/ActualityController.php]: You should not consider ids to sort your data by creation date. You should have a creation date field

File: src/Controller/ActualityController.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+     * @Route("/actualites/{id}/edit", name="actuality_edit")
+     */
+    public function editArticles(Request $request, Article $article): Response
+    {
+        /*$article->getImage()->removeElement($image);
+        $image->getArticles()->removeElement($article);*/
+        $form = $this->createForm(ArticleFormType::class, $article);
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $this->getDoctrine()->getManager()->flush();
```
Comment (RomainClair) [inline src/Controller/ActualityController.php]: The symfony best practice is to use dependency injection to access your services

**Review by RomainClair — DISMISSED** [review]

File: src/Controller/ActualityController.php (hunk 1 above)
Comment (KevinRoig) [reply src/Controller/ActualityController.php]: Ok, modified.

File: src/Controller/ActualityController.php (hunk 2 above)
Comment (KevinRoig) [reply src/Controller/ActualityController.php]: Ok, modified.

**Review by AurelieFqt — APPROVED** [review]

File: src/Controller/GalleryController.php [hunk 3]
```diff
@@ -22,19 +22,24 @@ public function showImages(ImageRepository $imageRepository): Response
     {
         $images = $imageRepository->findBy(
             ['categorie' => ['chats', 'chiens', 'oiseaux', '4', '5', '6', '7', '8', '9', '10']],
+            ['id' => 'DESC'],
```
Comment (RomainClair) [inline src/Controller/GalleryController.php:25]: You should probably avoid sorting by id

File: src/Controller/GalleryController.php [hunk 4]
```diff
@@ -22,19 +22,24 @@ public function showImages(ImageRepository $imageRepository): Response
...
      */
     public function showImagesByCategorie(string $categorie, ImageRepository $imageRepository): Response
     {
-        $images = $imageRepository->findBy([
-            'categorie' => $categorie
-        ]);
-        return $this->render('gallery/design.html.twig', ['images' => $images, 'categorie' => $categorie]);
+        $images = $imageRepository->findBy(
+            ['categorie' => $categorie],
+            ['id' => 'DESC']
```
Comment (RomainClair) [inline src/Controller/GalleryController.php:40]: Again

**Review by RomainClair — APPROVED** [review]

**Pull request merged**